src/components/ListTemplate/ColumnTemplates/MatchColumn.js

```diff
@@ -27,14 +27,19 @@ export const MatchColumn = memo(({
 }) => {
   const {
     existingRecordType,
-    existingStaticValueType,
     field,
-    fieldMarc,
-    fieldNonMarc,
+    matchDetails,
   } = record;
 
   const fieldSource = (field || existingRecordType || '').replace(/_/g, ' ');
-  const fieldMatched = (fieldMarc || fieldNonMarc || existingStaticValueType || '').replace(/_/g, ' ');
+  const matchRule = matchDetails[0];
+  const fieldsMatched = matchRule.existingMatchExpression.fields.flatMap(item => item.value);
+
+  if (document.dir === HTML_LANG_DIRECTIONS.RIGHT_TO_LEFT) {
+    fieldsMatched.reverse();
+  }
+
+  const fieldMatched = fieldsMatched.join(fieldsMatched.length > 1 ? '.' : '').replace(/_/g, ' ');
```

Comment:
```suggestion
  const fieldMatched = fieldsMatched.join('.').replace(/_/g, ' ');
```
I think it can be simplified.

Reply:
Done